index.md

```diff
@@ -3,4 +3,16 @@ layout: default
 title: Home
 ---
 
-Content coming soon.
+Tackling engineering design problems using tools from convex optimization and computer science.
+
+### Recent posts
+<div>
```

Comment:
done. I'm going to squash all these oneliner commits and force push to this branch -- standby